ISISComputingGroup/JSON_bourne
# Minor updates to make deploying site work

Fixes https://github.com/ISISComputingGroup/IBEX/issues/1680

To test:
* Go to the Larmor dashboard via http://dataweb.isis.rl.ac.uk/Dashboards/default.htm 
* Confirm this site works in a number of browsers 
* Confirm this works offsite (e.g. via your mobile?)
* Confirm that when an instrument restarts the webpage reconnects (you can get to demo's dashboard via http://dataweb.isis.rl.ac.uk/IbexDataweb/default.html?instrument=demo)
